deob-ast/src/main/kotlin/org/openrs2/deob/ast/AstDeobfuscatorModule.kt

```diff
@@ -33,24 +35,41 @@ public object AstDeobfuscatorModule : AbstractModule() {
             .`in`(Scopes.SINGLETON)
 
         val binder = Multibinder.newSetBinder(binder(), Transformer::class.java)
-        binder.addBinding().to(UnencloseTransformer::class.java)
-        binder.addBinding().to(NegativeLiteralTransformer::class.java)
-        binder.addBinding().to(ComplementTransformer::class.java)
-        binder.addBinding().to(NotTransformer::class.java)
-        binder.addBinding().to(CharLiteralTransformer::class.java)
-        binder.addBinding().to(IfElseTransformer::class.java)
-        binder.addBinding().to(TernaryTransformer::class.java)
-        binder.addBinding().to(BinaryExprOrderTransformer::class.java)
-        binder.addBinding().to(AddSubTransformer::class.java)
-        binder.addBinding().to(IdentityTransformer::class.java)
-        binder.addBinding().to(BitMaskTransformer::class.java)
-        binder.addBinding().to(HexLiteralTransformer::class.java)
-        binder.addBinding().to(ValueOfTransformer::class.java)
-        binder.addBinding().to(NewInstanceTransformer::class.java)
-        binder.addBinding().to(IncrementTransformer::class.java)
-        binder.addBinding().to(ForLoopConditionTransformer::class.java)
-        binder.addBinding().to(RedundantCastTransformer::class.java)
-        binder.addBinding().to(GlTransformer::class.java)
-        binder.addBinding().to(EncloseTransformer::class.java)
+
+        if (Files.notExists(Paths.get("share", "deob", "ast.transformers.list"))) {
```

Comment:
Have discussed a different approach via Discord and done an initial implementation for the deobfuscator here:

https://github.com/openrs2/openrs2/commit/e6f7a40f6eaae56de9715dd27f8f5585ab2525b7

Reply:
Yep, I like that approach much more. I’ll remove the changes around that from this PR’s scope and maybe just focus this on module loading…